### PR TITLE
redesign articles pubs page

### PR DESCRIPTION
**Pull request by andresgnlez**

### CODE REVIEW

File: app/src/components/ArticlesPublications.jsx [hunk 1]
```diff
@@ -23,26 +51,40 @@ class ArticlesPublications extends Component {
...
+            </h2>
+            <div className={PubsArticlesLayoutStyles.grid}>
+              {publications}
+            </div>
           </section>
         );
       }
     }
 
     if (articlesPublications && articlesPublications.publications) {
```
Comment (tiagojsag) [inline app/src/components/ArticlesPublications.jsx]: checking for publications, using articles (same thing above)

File: app/src/components/ArticlesPublications.jsx [hunk 2]
```diff
@@ -23,26 +51,40 @@ class ArticlesPublications extends Component {
     let pageContent = [];
 
     if (articlesPublications && articlesPublications.articles) {
-      const articles = articlesPublications.articles.map(this.renderElement);
+      const publications = articlesPublications.publications.map(this.renderElement);
 
-      if (articles && articles.length) {
```
Comment (tiagojsag) [inline app/src/components/ArticlesPublications.jsx]: out of curiosity, why did you change these?

Comment (andresgnlez) [inline app/src/components/ArticlesPublications.jsx]: The name? Yes, it less confuse than having two `publications` variables, in my opinion.

Comment (tiagojsag) [inline app/src/components/ArticlesPublications.jsx]: no, in here we had articles and you replaced with publications. In line 76, we had publications and you replaced with articles. code is the same, you just changed the order for some reason

Comment (andresgnlez) [inline app/src/components/ArticlesPublications.jsx]: Publications come first in the design.

**Comment by tiagojsag** [comment]
Tested it and it looks good

File: app/src/components/Shared/Rhombus.jsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ class Rhombus extends Component {
     const direction = this.props.direction ? RhombusStyles[this.props.direction] : '';
     const color = this.props.color ? RhombusStyles[this.props.color] : '';
 
-    return `${RhombusStyles['c-link-to']} ${direction} ${color}`;
+    return `${RhombusStyles['c-rhombus']} ${direction} ${color}`;
```
Comment (nerik) [inline app/src/components/Shared/Rhombus.jsx]: The whole method body could be replaced by:
```
classnames(RhombusStyles['c-rhombus'], RhombusStyles[this.props.direction], RhombusStyles[this.props.color]);
```

File: app/styles/components/c-publication-article.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+  > .title a {
+    color: $color-9;
+    font-family: $font-family-1;
+    font-size: $font-size-x-medium;
+    font-weight: $font-weight-light;
+    line-height: 1.4;
+  }
+
+  > .description p {
```
Comment (nerik) [inline app/styles/components/c-publication-article.scss]: Definitely not critical on this page, but avoid styling tags directly, performance is bad.

File: app/styles/components/c-publication-article.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    font-family: $font-family-2;
+    font-size: $font-size-small;
+    font-weight: $font-weight-bold;
+    letter-spacing: -.1px;
```
Comment (nerik) [inline app/styles/components/c-publication-article.scss]: Why do you need this?

File: app/styles/layout/l-publications-articles.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+  .section-title {
+    margin: 0  0 30px;
+
+    @media #{$mq-tablet} {
+      margin: 0  0 60px;
+    }
+  }
+
+  section {
```
Comment (nerik) [inline app/styles/layout/l-publications-articles.scss]: Re https://github.com/Vizzuality/GlobalFishingWatch/pull/164/files#r75878183

File: app/styles/layout/l-publications-articles.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    padding: 55px 0 65px;
+    position: relative;
+
+    &::after {
```
Comment (nerik) [inline app/styles/layout/l-publications-articles.scss]: Why not applying a border-bottom instead?
(you'd need to have `<div className={AppStyles.wrap}>` inside `section`s though)
Not sure about this.

**Comment by nerik** [comment]
Looks good, just a few comments.
I think at some point we'll have to split ArticlesPublications.jsx into subcomponents.

**Comment by andresgnlez** [comment]
Yeah, I agree with you on spliting both kind of publications.

File: app/styles/layout/l-publications-articles.scss (hunk 7 above)
Comment (andresgnlez) [inline app/styles/layout/l-publications-articles.scss]: I use `::after` because lines have to occupy the whole screen width and you can't do that with `border-bottom`. If you have any suggestion about this, it's welcome :)

I don't use `wrap` inside sections, I only use once and its surrounding them :/

File: app/styles/components/c-publication-article.scss (hunk 5 above)
Comment (andresgnlez) [inline app/styles/components/c-publication-article.scss]: Because those styles are quite different from usual ones. Do you mean this?

Comment (nerik) [inline app/styles/components/c-publication-article.scss]: I mean `letter-spacing: -.1px;`, just wondering what's that for.

File: app/styles/layout/l-publications-articles.scss (hunk 7 above)
Comment (nerik) [inline app/styles/layout/l-publications-articles.scss]: Nevermind then.
